src/main/java/at/hannibal2/skyhanni/features/mining/TunnelsMaps.kt

```diff
@@ -198,6 +207,7 @@ object TunnelsMaps {
         if (!isEnabled()) return
         if (event.clickedButton != 1) return
         clickTranslate[event.slotId]?.let {
+            isCommission = false
```

Comment:
This should be true

Reply:
Originally I wanted to remove that line but forgot.

What is the difference between this click detection and the normal open inventory detection? Since either the gold gets reset, or we need to wait for the next inventory update to see the next goal anyway.
so there should be nothing to change "tunnel maps"-wise when clicking the on the commission item. or am i missing something?

Reply:
We have 3 modes of operation:
 * Manuel (using our gui to set any goal)
 * Automatic (which grabs the goal at inventory open)
 * Select (You can click a commission to chose to follow it instead)
 
 And this is option 3.

Reply:
can you unfollow a automatic commission by clicking this way as well?

Reply:
The last action has the highest priority.